README.md

```diff
@@ -38,28 +40,351 @@ testing and development.
 
 ## Installation
 
-This guide assumes you have NGINX/Apache installed, along with the dependencies.
-Those will not be covered in these early docs.
+Install Git + Download Scripts
+```sh
+$ sudo apt install git
+```
+
+Install sury.org package source
+```sh
+$ sudo apt-get update
+$ sudo apt-get -y install apt-transport-https lsb-release ca-certificates
+$ sudo wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg
+$ sudo sh -c 'echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php.list'
+$ sudo apt-get update
+```
+
+Install Erlang Solutions repo
+```sh
+$ sudo wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && ${SUDO} dpkg -i erlang-solutions_1.0_all.deb        
+$ sudo apt-get update
+```
+
+Install Elixir   
+```sh
+$ sudo apt-get install -y erlang-dev erlang-xmerl erlang-parsetools elixir
+```
+
+Install Node.js 10.x
+```sh
+$ sudo wget -qO- https://deb.nodesource.com/setup_10.x | ${SUDO} -E bash -
+$ sudo apt-get install -y nodejs build-essential
+```
+
+Install Erlang Solutions repo
+```sh
+$ sudo wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && ${SUDO} dpkg -i erlang-solutions_1.0_all.deb        
+$ sudo apt-get update
+```
+
+Install Elixir   
+```sh
+$ sudo apt-get install -y erlang-dev erlang-xmerl erlang-parsetools elixir
+```
+
+Install Node.js 10.x
+```sh
+$ sudo wget -qO- https://deb.nodesource.com/setup_10.x | ${SUDO} -E bash -
+$ sudo apt-get install -y nodejs build-essential
+```
+
+Install Yarn
+```sh
+$ sudo apt-get update && ${SUDO} apt-get install yarn
+```
+
+Install htop, bmon, mc and misc tools required by pixelfed
```

Comment:
`htop`, `bmon` and `mc` are not required by pixelfed

Reply:
Updated